### ISSUE TITLE
Integrate application with loggly

### ISSUE THREAD

**Issue by hdrury1** [elifesciences/elife-xpub#692]
Logging has now been configured, but final integration work needs to be completed.

**Comment by diversemix** [comment]
@giorgiosironi  - Am I correct in thinking that you are the only one who can do this ticket (now we have configured the logs for xpub)

**Comment by giorgiosironi** [comment]
It's all formula changes but very similar to what we already have so best for me to open some PRs about it.

**Comment by giorgiosironi** [comment]
Testing on staging:
```
Oct 10 17:08:45 staging--xpub.elifesciences.org syslog-ng[1122]: Follow-mode file source not found, deferring open; filename='/srv/elife-xpub/var/logs/*.log'
```
The file is now there, so it may be picked up. I thought it was there already however, from previous deployments.

**Comment by giorgiosironi** [comment]
Manually tweaked so that:
https://elifesciences.loggly.com/search#terms=&from=2018-10-11T07:22:30.391Z&until=2018-10-11T07:32:30.391Z&source_group=&filter=syslog.host;staging--xpub.elifesciences.org&filter=syslog.appName;elife-xpub
works. The `*.log` path to match files doesn't seem to work, though I've seen it elsewhere.

**Comment by giorgiosironi** [comment]
In fact, the same filter from [`profiles-formula`](https://github.com/elifesciences/profiles-formula/blob/master/salt/profiles/config/etc-syslog-ng-conf.d-profiles.conf) doesn't seem to work too.

**Comment by giorgiosironi** [comment]
[We should use the `wildcard-file` source](https://www.syslog-ng.com/technical-documents/doc/syslog-ng-open-source-edition/3.16/administration-guide/18#TOPIC-956450]).

**Comment by giorgiosironi** [comment]
But it's only available in 3.10 and we are on 3.5.6 on Ubuntu 16.04.

**Comment by giorgiosironi** [comment]
`3.5.6` is the latest available on [16.04](https://packages.ubuntu.com/source/xenial/syslog-ng), 18.04 would have [3.13.2-3](https://packages.ubuntu.com/source/bionic/syslog-ng) but has other problems like New Relic infrastructure lack of support. Will put it with a fixed single path then.

**Comment by giorgiosironi** [comment]
Working in [prod](https://elifesciences.loggly.com/search#terms=&from=2018-10-11T08:34:32.971Z&until=2018-10-11T08:44:32.971Z&source_group=&filter=syslog.appName;elife-xpub)
